site/content/contribute/getting-started/contribution-checklist.md

```diff
@@ -5,30 +5,29 @@ weight: 1
+Thanks for your interest in contributing to Mattermost! Come join our [Contributors community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team.
 
 To help with translations, [see the localization process](https://docs.mattermost.com/developer/localization.html).
 
 Follow this checklist for submitting a pull request (PR):
 
 1. You've signed the [Contributor License Agreement](http://www.mattermost.org/mattermost-contributor-agreement/), so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true).
  - If you've included your mailing address in the signed [Contributor License Agreement](https://www.mattermost.org/mattermost-contributor-agreement/), you may receive a [Limited Edition Mattermost Mug](https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143) as a thank you gift after your first pull request is merged.
-2. Your ticket is a help wanted GitHub issue for one of the Mattermost project you're contributing to.
+2. Your ticket is a Help Wanted GitHub issue for the Mattermost project you're contributing to.
     - If not, follow the process [here](/contribute/getting-started/contributions-without-ticket).
 3. Your code follows the [Mattermost Style Guide](http://docs.mattermost.com/developer/style-guide.html).
-4. Code thoroughly tested, including appropriate unit tests, [end-to-end tests for webapp](/contribute/webapp/end-to-end-tests/) and manual testing.
+4. Your code is thoroughly tested, including appropriate unit tests, [end-to-end tests for webapp](/contribute/webapp/end-to-end-tests/), and manual testing.
 5. If applicable, user interface strings are included in localization files:
     - [../mattermost-server../en.json](https://github.com/mattermost/mattermost-server/blob/master/i18n/en.json)
     - [../mattermost-webapp../en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)
     - [../mattermost-mobile../en.json](https://github.com/mattermost/mattermost-mobile/blob/master/assets/base/i18n/en.json)
 
     5.1 In the webapp repository run `make i18n-extract` to generate the new/updated strings.
-6. PR is submitted against `master` branch.
-7. PR title begins with the JIRA or GitHub Ticket ID (e.g. `MM-394` or `GH-394`) and summary template is filled out.
-8. Once submitted, the automated build process must pass for the PR.
-9. If your PR adds or changes a RESTful API endpoint, please update the [API documentation](https://github.com/mattermost/mattermost-api-reference).
+6. The PR is submitted against `master` branch in a new fork (as prompted in the GitHub UI).
```

Comment:
I'm not sure if this is correct. The PR should be submitted to `master` on upstream _from_ the fork. Could you make this more clear?

Reply:
You're absolutely right - I meant to say submitted to master in a new branch but that assumes the contributor has write-access to the repo. If not, they'd need to create a fork and then submit from the fork. 

As we don't mention forking in this content there are two options:
1. Add a step regarding forking before getting started with contributions and then include those steps as a sub-step of this. Not sure if forking is just generally assumed though in which case it'd be redundant. 

2. Add a step regarding forking for contributors and note that core-committers can work off a branch from master. Then divide this into the same sub-steps as above?

Let me know what you think? :)

Reply:
I would go with 2. Core-comitters are not the prime audience for this checklist as they are (in general) knowable about Git and know what to do.

Reply:
I've edited it slightly based on two things:
1. The steps for contributing to the various projects do indicate that the contributor needs to make a fork. 
2. I am working on restructuring this documentation and will add that to the overall "Getting Started" content for additional clarity.